### ISSUE TITLE
Notification support for bazel commands 

### ISSUE THREAD

**Issue by brandonNguyenSFDC** [bazelbuild/intellij#6160]
### Description of the feature request:

When actions by the plugin complete such as sync or configured commands an intellij notification is created

### Which category does this issue belong to?

Intellij

### What underlying problem are you trying to solve with this feature?

Often my bazel actions take a while to complete and I would like to have some way to be notified when their complete other than visually checking the status of the command.

### What operating system, Intellij IDE and programming languages are you using? Please provide specific versions.

macOS Sonoma 14.3.1 Intellij 2023.3.3 Java

**Comment by mai93** [comment]
IIRC this used to work for sync operation. @tpasternak if you can task a look

**Comment by brandonNguyenSFDC** [comment]
@tpasternak 

> So just tried it briefly notifications after `Sync` do work (if IntelliJ window is out of focus)
> 
> https://github.com/bazelbuild/intellij/blob/fec469bed8c636c3cf37a3c224cd2a35b3e80495/base/src/com/google/idea/blaze/base/sync/SyncPhaseCoordinator.java#L711

I don't believe that this notification is customizable to make sound as I don't see it show up in the notifications tool window. Ideally all the notifications that the bazel plugin make would show up there.
